src/hgvs/utils/reftranscriptdata.py

```diff
@@ -20,14 +20,21 @@ def __init__(self, hdp, tx_ac, pro_ac):
 
         # coding sequences that are not divisable by 3 are not yet supported
         tx_seq_to_translate = tx_seq[cds_start - 1 : cds_stop]
+        if translation_table == TranslationTable.vertebrate_mitochondrial:
+            if len(tx_seq_to_translate) % 3 == 1 and tx_seq_to_translate[-1] == "T":
```

Comment:
@b0d0nne11 : I'm not familiar with this mito translation rule. It appears to me that you're effectively creating an explicit stop codon if it's incomplete. (TAA = UAA is one of 4 valid mito stop codons)

Is that right?